### ISSUE TITLE
Coordinate System Feature Request

### ISSUE THREAD

**Issue by dunyakilavuz** [godotengine/godot#16088]
Hi everybody, I was experimenting with Godot 3 and implementing some of my old unity projects into the engine to see how Godot handles things differently. Most of the features were easy to learn and when something is handled differently it was easy to see why it has to be that way. Except this thing about the coordinate system.
The Unity coordinate system uses X+ as right direction, Y+ as up direction and finally Z+ as forward direction (Left-Handed Coordinates.) . Demonstration in image below;

![unitycoord](https://user-images.githubusercontent.com/9383042/35450857-5bc6e66a-02d2-11e8-84fb-c509c506da02.JPG)
Now the Godot coordinate system uses X+ as right direction, Y+ as up direction and Z- as forward direction (Right-Handed Coordinates) . Again demonstration in image below;

![godotcoord](https://user-images.githubusercontent.com/9383042/35451353-fa5f6616-02d3-11e8-9199-1e709e70ae9c.JPG)

Now when I try to move an object along the Z+ direction in Godot, the object moves in the opposite direction, similarly when I want to rotate an object around a point it rotates around in the opposite direction of what I intended.
In conclusion I think adding a Left-Handed Coordinate system option to the engine would help a lot of developers like me who are new to Godot and familiar with Unity's coordinate system.
I am sorry if it was already a feature, and sorry for bad editing. Thanks for your attention!

**Comment by Zireael07** [comment]
It's not already a feature, I believe it was already requested and it was decided it won't happen (too many things would have to be changed)

**Comment by reduz** [comment]
No, this is not selectable on any application. It's way too much work for no gain.
Unity is different than Unreal, which is different to Blender, Maya, etc. You can't expect this to be an universal standard.

Godot was born on Linux+OpenGL and it uses the OpenGL standard (like Maya) and it's proud of it's roots.

**Comment by Thaina** [comment]
> No, this is not selectable on any application. It's way too much work for no gain. Unity is different than Unreal, which is different to Blender, Maya, etc. You can't expect this to be an universal standard.
> 
> Godot was born on Linux+OpenGL and it uses the OpenGL standard (like Maya) and it's proud of it's roots.

Then you are wrong. OpenGL is actually internally left hand coordinate and it was misused by architect

**Comment by cyberfrogg** [comment]
@Thaina 
Up

**Comment by akien-mga** [comment]
This was closed 6 years ago and is not up for discussion or change.
